Source/AssetRipper.GUI.Electron/Pages/Assets/View.cshtml

```diff
@@ -28,6 +30,10 @@
 			<button class="@TabHelpers.GetTabClassSet(imageTabDisabled)" id="nav-image-tab" data-bs-toggle="tab" data-bs-target="#nav-image" type="button" role="tab" aria-controls="nav-image" aria-selected="false" aria-disabled="@imageTabDisabled.ToString().ToLower()">
 				@Localization.AssetTabImage
 			</button>
+			<button class="@TabHelpers.GetTabClassSet(fontTabDisabled)" id="nav-font-tab" data-bs-toggle="tab" data-bs-target="#nav-font" type="button" role="tab" aria-controls="nav-font" aria-selected="false" aria-disabled="@fontTabDisabled.ToString().ToLower()">
+				Font
+				@* TODO: @Localization.AssetTabFont *@
```

Comment:
Don't worry about this. I'm thinking of merging the tabs into a single one with a name like `Multimedia` or `Preview`.